fury/actors/peak.py

```diff
@@ -0,0 +1,343 @@
+from fury.colormap import boys2rgb, colormap_lookup_table, orient2rgb
+from fury.shaders import attribute_to_actor, load, shader_to_actor
+from fury.utils import apply_affine, numpy_to_vtk_colors, numpy_to_vtk_points
+from vtk.util import numpy_support
+
+
+import numpy as np
+import vtk
+
+
+class PeakActor(vtk.vtkActor):
+    def __init__(self, directions, indices, values=None, affine=None,
+                 colors=None, lookup_colormap=None, linewidth=1):
+        if affine is not None:
+            w_pos = apply_affine(affine, np.asarray(indices).T)
+
+        valid_dirs = directions[indices]
+
+        points_array = []
+        centers_array = []
+        diffs_array = []
+        for idx, center in enumerate(zip(indices[0], indices[1], indices[2])):
+            if affine is None:
+                xyz = np.array(center)
+            else:
+                xyz = w_pos[idx, :]
+            valid_peaks = np.nonzero(
+                np.abs(valid_dirs[idx, :, :]).max(axis=-1) > 0.)[0]
+            for pdir in valid_peaks:
+                if values is not None:
+                    pv = values[center][pdir]
+                else:
+                    pv = 1.
+                point_i = directions[center][pdir] * pv + xyz
+                point_e = -directions[center][pdir] * pv + xyz
+                diff = point_e - point_i
+                points_array.append(point_e)
+                points_array.append(point_i)
```

Comment:
if you keep a list, you cando `points_array += [point_e, point_i]`

Reply:
Changed to numpy empty array.